test/lit/passes/separate-data-segments.wast

```diff
@@ -0,0 +1,29 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+;; RUN: wasm-opt %s --separate-data-segments=%t.data --pass-arg=global-base@1024 -S -o - | filecheck %s
+;; RUN: cat %t.data | filecheck %s --check-prefix=CHECK-DATA
+
+(module
+  (memory 1 1)
+
+  ;; CHECK:      (type $0 (func))
+
+  ;; CHECK:      (memory $0 1 1)
+
+  ;; CHECK:      (func $foo
+  ;; CHECK-NEXT:  (call $bar)
+  ;; CHECK-NEXT: )
+  (func $foo
+    (call $bar)
+  )
+
+  ;; CHECK:      (func $bar
+  ;; CHECK-NEXT:  (call $foo)
+  ;; CHECK-NEXT: )
+  (func $bar
+    (call $foo)
+  )
+
+  (data (i32.const 1024) "hello world\n")
+)
+
+;; CHECK-DATA: {{^}}hello world{{$}}
```

Comment:
Is there some way ti can keep this hand-written check for the `cat %t.data | filecheck %s --check-prefix=CHECK-DATA` command above while still using `update_lit_checks.py` to update the rest of the checks?

Reply:
I don't think so - each test file is either automatic or not. Might need separate files then. But @tlively might have a better idea?

Reply:
No, unfortunately the updater script does not support mixing hand-written and generated checks. We could potentially add that as a feature, though. I don't think it would be too hard to add a flag telling the updater to ignore preserve checks with a particular `CHECK` prefix. OTOH, the updater script is already rather complicated.